modules/objdetect/misc/python/test/test_qrcode_detect.py

```diff
@@ -13,6 +13,21 @@
 class qrcode_detector_test(NewOpenCVTests):
     def test_detect_and_decode(self):
         img = cv.imread(os.path.join(self.extraTestDataPath, 'cv/qrcode/link_ocv.jpg'))
+        self.assertFalse(img == None)
```

Comment:
> == None

In Python world checks should look like:
- `a is None`
- `a is not None`

Reply:
@alalek Which way is better img is None or np.shape(img) is () ?